### PR TITLE
Add status page to the Author application.

### PR DESCRIPTION
**Pull request by samiwel**
### What is the context of this PR?
This PR introduces a new `/#/status` route for the author application.
A `Status` component has been introduced which outputs simple text with a status of "OK" and the Git SHA-1 has if supplied.

### How to review 
Build a docker container on this branch using the following command:
```docker build -t onsdigital/eq-author:latest --build-arg APPLICATION_VERSION=$(git rev-parse HEAD) .```
Run the newly built author image pointing at a valid API url e.g.
```docker run -ti -e REACT_APP_API_URL=http://docker.for.mac.localhost:4000/graphql 127328c9b756``` (Replace *127328c9b756* with the image Id for onsdigital/eq-author:latest, which can be found by running `docker images`.
Navigate to the application running on port 3000. Browse to the `/#/status` endpoint.
You should see status OK and the Git SHA-1 hash for version.

### CODE REVIEW

**Comment by samiwel** [comment]
[Add status pages (3)](https://trello.com/c/R6n1aCJP/250-add-status-pages-3)

File: .travis.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ script:
 after_success:
   - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
   - export TAG=`if [ "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]; then echo "latest"; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi`
-  - docker build -t onsdigital/eq-author:$TAG -f Dockerfile .
+  - docker build -t onsdigital/eq-author:$TAG --build-arg APPLICATION_VERSION=$(git rev-parse HEAD) -f Dockerfile .
```
Comment (jonnyshaw89) [inline .travis.yml]: Im not sure if this is the best way to do this. Wont this just read the ENV_VAR from the client?

**Comment by WickyNilliams** [comment]
What is the purpose of this page @jonnyshaw89? Is it to test the deploy happened? Or test the app is "running" (i.e. JS is not crashing)?

**Comment by jonnyshaw89** [comment]
@WickyNilliams its mainly just a way to check is the app running and what version of the app is running

**Comment by WickyNilliams** [comment]
@jonnyshaw89, i think this now works as expected. The netlify deploy has the status page: https://deploy-preview-267--author.netlify.com/status.json

If you're happy, could you approve?

**Pull request merged**